make0x.py

```diff
@@ -13,7 +13,7 @@ def output_on_failure(cmd, cwd):
     )
     if ret.returncode:
         raise AssertionError('Command {!r} returned {}!\nOutput:\n{}'.format(
-            cmd, ret.returncode, ret.stdout,
+            cmd, ret.returncode, ret.stdout.decode(),
```

Comment:
skipping the decode here was ~partially intentional (non-UTF-8 code would crash while producing an error message instead of showing the error message)

Reply:
Yes, but the error output was really unreadable single line -- example:

```console
b"long error message            without new lines and with             white spacing all over the place"
```

This is why I added the decode - not sure if there is a better way to fix the output.